plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/TestMongoConnectorTest.java

```diff
@@ -956,8 +958,10 @@ public void testNullPredicates()
     }
 
     @Test
+    @Override
     public void testLimitPushdown()
     {
+        super.testLimitPushdown();
```

Comment:
nit: It would be nice to separate a test method instead of overriding in my opinion. Same for `BaseJdbcConnectorTest`.